digdag-docs/src/command_executor.md

```diff
@@ -0,0 +1,141 @@
+# Command Executor
+`Command Executor` provides a functionality to run operators in various environment like Docker.
+
+`sh>`, `py>` `rb>` support Command Executor.
+
+Supported environments are `AWS ECS (Elastic Container Service)`, `Docker`, and local.
+`Kubernetes` is under development.
+
+For example, if you define a task with `sh>`, the task runs in local. If you add configuration for Docker, the task is executed in a docker container.
+You can switch environment to run a task without changing task definition.
+
+Currently, ECS is default Command Executor.
+If there is no valid configuration for ECS, fallback to Docker. 
+If there is no valid configuration for Docker, fallback to local.
+
+## ECS
+### Setup
+The following is an example configuration for ECS Command Executor.
+
+```
+agent.command_executor.ecs.name = digdag-test
+
+agent.command_executor.ecs.digdag-test.access_key_id = <ACCESS KEY>
+agent.command_executor.ecs.digdag-test.secret_access_key = <SECRET KEY>
+agent.command_executor.ecs.digdag-test.launch_type = FARGATE
+agent.command_executor.ecs.digdag-test.region = us-east-1
+agent.command_executor.ecs.digdag-test.subnets = subnet-NNNNN
+agent.command_executor.ecs.digdag-test.max_retries = 3
+
+agent.command_executor.ecs.temporal_storage.type = s3
+agent.command_executor.ecs.temporal_storage.s3.bucket = <Bucket>
+agent.command_executor.ecs.temporal_storage.s3.endpoint = s3.amazonaws.com
+agent.command_executor.ecs.temporal_storage.s3.credentials.access-key-id = <ACCESS KEY>
+agent.command_executor.ecs.temporal_storage.s3.credentials.secret-access-key = <SECRET KEY>
+```
```

Comment:
Regarding `access-key-id` and `secret-access-key`, we should mention about the fallback to commonly applied values (default vlaues) for all ECS clusters.

https://github.com/treasure-data/digdag/blob/cb35cf213c5afe896f3b385ad1ceb78ff12f6dec/digdag-standards/src/main/java/io/digdag/standards/command/ecs/EcsClientConfig.java#L78-L88

Reply:
@serihiro Thank you for your review.
The description focuses on the minimum/basic configuration to start ECS Command Executor.
If we describes about default `__default_config__`, it is better to add a sub-section "multiple cluster setting".
But I don't have enough time to write them and skip it. 
What do you think it ?
If you think we need to add them, you can create additional PR for it.

Reply:
Then I will add it to another PR later. Thanks.